pipelines/internal/commands/run/run.go

```diff
@@ -180,30 +181,50 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 	}
 	fmt.Printf("%s\n", encoded)
 
+	if *attempts == 0 {
```

Comment:
This should probably move to either 
(1) before building the request or
(2) just before starting the attempts loop